nengo/neurons.py

```diff
@@ -241,11 +245,19 @@ class RectifiedLinear(NeuronType):
 
     probeable = ('rates',)
 
+    amplitude = NumberParam('amplitude', low=0, low_open=True)
+
     def __init__(self, amplitude=1):
         super(RectifiedLinear, self).__init__()
 
         self.amplitude = amplitude
 
+    @property
+    def _argreprs(self):
+        args = []
+        self._add_argrepr(args, 'amplitude', 1)
```

Comment:
Could we automate this (like we do in e.g. https://github.com/nengo/nengo/blob/master/nengo/params.py#L495), rather than having to remember to update this list whenever we add a parameter or change a default?

Reply:
The tests that @hunse added in #1472 will notice if you forget to update this list after adding a parameter / changing a default. So I don't think forgetting to do this will be a problem.